### PR TITLE
DLTP-601 OSF Integration Bugs

### PR DESCRIPTION
**Pull request by msuhovec**
#49

### CODE REVIEW

File: tasks/get-from-osf [hunk 1]
```diff
@@ -21,9 +21,11 @@ fi
 # Iterate through urls
 for this_project  in $(jq --raw-output .project_identifier < $osf_projects ); do
 
-  package_name=$( basename $this_project)
+  package_name=${this_project}
+  package_url="https://api.osf.io/v2/registrations/${package_name}/"
```
Comment (dbrower) [inline tasks/get-from-osf]: will this work when the item is not a registration?

File: tasks/osf-to-rof [hunk 2]
```diff
@@ -21,4 +21,10 @@ if !  bundle exec osf_to_rof --project_file $osf_projects --package_dir ${JOBPAT
        exit 1
 fi
 
+#If ROF construction succeeds, copy package file into top workdir, where subsequent ingest steps will expect it.
+if ! cp -f  "${JOBPATH}/FROM_OSF/${project_id}.tar.gz" "${JOBPATH}/${project_id}.tar.gz"; then
```
Comment (dbrower) [inline tasks/osf-to-rof]: Would a `mv` be better? Is there a reason to copy it?

**Comment by msuhovec** [comment]
1)  When the tool supports projects, and we update the jar, we'll change
this to either use a new path, or to try both the registratuins and nodes
routes

2) Preserving the contents of FROM_OSF seemed consistent with what we do
elsewhere for FROM_FEDORA and TO_TAPE

**Comment by dbrower** [comment]
1) fair enough. we will update it when the jar supports it
2) change it to a move. The bigger consideration is to not copy data unnecessarily. For example arrange-files moves data files into the TO_TAPE directory.

**Pull request merged**